sql.txt

```diff
@@ -1,14 +1,39 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country
+FROM Customers
+ORDER BY Country
+LIMIT 20
+OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT *
+FROM Customers
+WHERE ContactName LIKE "A%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT *
+FROM OrderDetails
+WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (SupplierID, CategoryID)
+VALUES (1, 2), (3, 4), (5, 6);
+
```

Comment:
you haven't added any info about the product itself (name, qty, price). It should look like:
INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Gum", 2, 5, "5 pack", 0.60);